src/thunderbots/software/grsim_communication/grsim_command_primitive_visitor.cpp

```diff
@@ -154,6 +154,13 @@ void GrsimCommandPrimitiveVisitor::visit(const PivotPrimitive &pivot_primitive)
 void GrsimCommandPrimitiveVisitor::visit(const DribblePrimitive &dribble_primitive)
 {
     // TODO: https://github.com/UBC-Thunderbots/Software/issues/107
+
+    float kickPower = dribble_primitive.isSmallKickAllowed() ? 5.0f : 0.0f;
```

Comment:
Where is `5.0f` coming from here? We should probably make this a named constant, and it should probably go in the `constants.h` file (though that would depend on if we'll use this constant elsewhere).